src/main/java/by/georgprog/epicmusicstore/controllers/client/AlbumController.java

```diff
@@ -0,0 +1,62 @@
+package by.georgprog.epicmusicstore.controllers.client;
+
+import by.georgprog.epicmusicstore.dto.AlbumDto;
+import by.georgprog.epicmusicstore.exeption.forbidden.ObtainingDataException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.AlbumNotFoundException;
+import by.georgprog.epicmusicstore.exeption.unauthorized.UserNotFoundException;
+import by.georgprog.epicmusicstore.service.album.AlbumService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@RequestMapping("users/{userId}/albums")
+@RequiredArgsConstructor
+public class AlbumController {
+
+    private final AlbumService albumService;
+
+    @GetMapping
+    public ResponseEntity<List<AlbumDto>> getAlbumsWithPicture() throws UserNotFoundException {
+        return new ResponseEntity<>(albumService.findAllOwnerAlbums(), HttpStatus.OK);
+    }
+
+    @GetMapping("/{albumId}")
+    public ResponseEntity<AlbumDto> getAlbum(@PathVariable Long albumId) throws AlbumNotFoundException,
```

Comment:
You may make these exceptions extend RuntimeException, to not specify them in throws block